zerver/tests/test_events.py

```diff
@@ -418,6 +418,7 @@ def test_stream_send_message_events(self) -> None:
                 propagate_mode,
                 False,
                 False,
+                False,
```

Comment:
We should probably do a refactor that allows us to make this request more readable (a series of Falses in a row is much worse than passing these parameters via kwargs where they are named wherever they are used)

Reply:
Done!